index.html

```diff
@@ -2,7 +2,108 @@
 <html>
   <head>
     <title>Mashup</title>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
+    <style>
+	    input {
+	    	margin: 10px; 
+	    }
+
+	    li{
+	    	list-style: none; 
+	    }
+
+	    .getimages {
+	    	width: 100px;
+	    	border : 1px solid black ;
+	    	cursor: pointer;
+	    	margin: 5px; 
+	    }
+
+	    .queriedImages {
+	    }
+
+    </style>
   </head>
+
   <body>
+ 	<input type = "text" class = "inputText"></input>
+ 	<button type = "submit"> Get images</button>
+ 	<button type = "reset"> Reset images</button>
+ 	<div class= "widget"></div>
+ 	<div class= "queriedImages"></div>
+
+
+  <script>
+	function loadImages(queryTag) {
+		$.ajax({
+		url: 'https://api.instagram.com/v1/tags/'+ queryTag +'/media/recent?client_id=ce95cb4e56c146c994457b48a839f6a8',
+		dataType: 'jsonp',
+		success: function(result) {
+		  for (var i = 0; i < result.data.length; i++) {
+		    var url = result.data[i].images.thumbnail.url;
+		    $('.queriedImages').append('<img class =\'EachImage\' src="' + url + '"/>');
+		  }
+		}
+		});
+	}
+
+	function searchArtist(query) {
+		$.ajax({
+			url:"https://api.spotify.com/v1/search?q="+ query +"&type=artist",
+			// Accept: application/json,
+			dataType: 'json',
+			success: function(result) {
+				console.log(result)
+				var name = result.artists.items[0].name
+				var id = result.artists.items[0].id
+				getArtistSongs(id)
+			}
+		});
+	}
+
+	function getArtistSongs(artistId) {
+		console.log("got the artistId:", artistId)
+		$.ajax({
+			url:"https://api.spotify.com/v1/artists/" + artistId + "/top-tracks?country=US",
+			dataType: 'json',
+			success: function(result) {
+		  		console.log("result", result)
+		  		
+		  		var track = result.tracks[0].id; 
+				widget(track)
+		  		// for (var i = 0; i < result.tracks.length; i++) {
+		  		// 	var track = result.tracks[i].id; 
+		  		// 	// console.log("track", track.name, track.id)
+		  		// 	widget(track)
+		  		// }
+
+			}
+		});
+	}
+
+	function widget(tracks) {
+		// var Playlist = $('<iframe src="https://embed.spotify.com/?uri=spotify:trackset:'+ tracks + 'frameborder="0" allowtransparency="true"></iframe>')
+		
+		var Playlist = $('<iframe src="https://embed.spotify.com/?uri=' + tracks +'" width="250" height="330" frameborder="0" allowtransparency="true"></iframe>')
+		var Playlist = $('<iframe class = \' widgetplay \' src="https://embed.spotify.com/?uri=spotify:track:' + tracks +'" frameborder="0" allowtransparency="true"></iframe>')
+
+		$(".widget").append(Playlist)	
+	}
+
+	$(document).ready(function() {	  	  
+		
+		$(":submit").click(function() {
```

Comment:
Good practice to limit this to a specific form, so that you don't actually catch submits for any other forms that might make their way into the page—would lead to some weird bugs.